deep-learning/_posts/2018-10-26-episodic_curiosity_through_reachability.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: review
+title: "Episodic curiosity through reachability"
+tags: deep-learning reinforcement
+author: "Antoine Théberge"
+cite:
+    authors: "Wei Yang, Xiaolong Wang, Ali Farhadi, Abhinav Gupta, Roozbeh Mottaghi5"
```

Comment:
Mottaghi5

nice name

Reply:
In fact, those are not the correct authors! lol

Reply:
Mottaghi5 is there in your review "Visual semantic navigation using scene priors"